app/players/player.py

```diff
@@ -0,0 +1,13 @@
+from abc import ABC, abstractmethod
+
+
+class Player(ABC):
+    @abstractmethod
+    def __init__(self, nickname: str) -> None:
+        self.nickname = nickname
+
+    def get_rating(self) -> None:
+        pass
+
+    def player_info(self) -> None:
+        pass
```

Comment:
I think these two methods `get_rating player_info` should be abstract)

Reply:
There are 2 method in Player class. But in Dwarf class and Elf class I can't define it, because I don't have any condition to do it. Of course I could be wrong, but I don't know how to fix it =(

Reply:
you can leave them abstract
actually should define an abstract method in a child class only if it's non-abstract. `Efl` and `Dwarf` are abstract classes so you can leave some of the parent's methods non-implemented